pages/identities.tsx

```diff
@@ -95,6 +95,12 @@ const Identities: NextPage = () => {
     execute();
   }
 
+  useEffect(()=>{
```

Comment:
Instead of redirecting the user, wrap your code from `pages/identities/[tokenId].tsx` in a component you can reuse in both files